### DISCUSSION TITLE
VSD for Shared Reading

### DISCUSSION THREAD

**Discussion by aetkie** [UNC-Project-Open-AAC/OS-DPI#175] [Show and tell]
The goal for this design was to create a VSD that simulates shared reading between an adult and a child. The hotspots, when activate, output comments that an adult might make during shared reading to draw the child's attention to the story context, model commenting, and elicit a response from the child. The symbols in the bottom navigation bar are examples of core vocabulary that the child might use to respond to the adult's comment or that the adult could use to model a response. 

<img width="539" alt="Screenshot 2023-03-15 at 8 24 09 PM" src="https://user-images.githubusercontent.com/113473933/225477412-6a4790c8-839b-4f82-99f7-4f03b40960f4.png">

Excel sheet shared to show how I arranged state variables and content:
[Tough Boris_VSD.xlsx](https://github.com/UNC-Project-Open-AAC/OS-DPI/files/10985552/Tough.Boris_VSD.xlsx)

OSDPI file:

[Tough_Boris_011723.osdpi.zip](https://github.com/UNC-Project-Open-AAC/OS-DPI/files/10985570/Tough_Boris_011723.osdpi.zip)

**Comment by sofia-bg** [comment]
@aetkie Awesome!!! I would love to check it out, but this file wouldn't import for me. I wonder if you just replace .osdpi with .zip instead of adding it to the end if that would work? Thanks!!

**Reply by gbishop** [reply]
She zipped the osdpi file. You'll need to unzip it and then it should work.

**Comment by gbishop** [comment]
That *should* work (where should is a big word...)

You can't simply rename it. I imagine that if you simply double click on it
(or whatever you mac people do to open something) it will offer to extract
that file that is contained inside the zip.

**Reply by sofia-bg** [reply]
on my machine if i export an .osdpi file, changing the extension name to .zip does convert to a zip file... its how I can take symbols someone else has used/uploaded... it is ok, I'll figure it out when my brain has more dedicated time to it and not mushy from a major manuscript revision... clearly user error over here.

**Comment by gbishop** [comment]
It doesn't "convert" it to a zip file because it already is a zip file. The
.osdpi file *is* a zip file, I just gave it a different suffix so people
wouldn't think they could import any arbitrary zip file.

Confusing, I know.

**Reply by sofia-bg** [reply]
well i suppose what i mean is that by changing the name i am able to then open the file on my desktop in a way that i cannot with the .osdpi extension.

**Comment by gbishop** [comment]
That is correct. The file that is giving you problems is a zip file
containing another zip file. You've got to unzip it to get to the inner zip
file.
